src/test/resources/outputs/redshift/basics/exclude.sql

```diff
@@ -1,115 +1,115 @@
--- -- Baseline query without `EXCLUDE`
--- --#[exclude-36]
--- SELECT "t"."a", "t"."b", "t"."c", "t"."d", "t"."e", "t"."f", "t"."g" FROM "default"."T_EXCLUDE_TOP_LEVEL" AS "t";
---
--- -- EXCLUDE single top-level column (no `t.a`)
--- --#[exclude-37]
--- SELECT "t"."b", "t"."c", "t"."d", "t"."e", "t"."f", "t"."g" FROM "default"."T_EXCLUDE_TOP_LEVEL" AS "t";
---
--- -- EXCLUDE multiple top-level columns (no `t.a` through `t.e`)
--- --#[exclude-38]
--- SELECT "t"."f", "t"."g" FROM "default"."T_EXCLUDE_TOP_LEVEL" AS "t";
---
--- -- EXCLUDE top-level columns with WHERE
--- --#[exclude-39]
--- SELECT "t"."c", "t"."d", "t"."e", "t"."f", "t"."g" FROM "default"."T_EXCLUDE_TOP_LEVEL" AS "t" WHERE "t"."a" AND ("t"."c" = 'remove');
---
--- -- EXCLUDE top-level columns with explicit SELECT list
--- --#[exclude-40]
--- SELECT "t"."c", "t"."d", "t"."e", "t"."f", "t"."g" FROM "default"."T_EXCLUDE_TOP_LEVEL" AS "t" WHERE "t"."a" AND ("t"."c" = 'remove');
---
--- -- EXCLUDE top-level with subquery
--- --#[exclude-41]
--- SELECT "subq"."a", "subq"."b", "subq"."c", "subq"."d", "subq"."e", "subq"."f", "subq"."g" FROM (SELECT "t"."a", "t"."b", "t"."c", "t"."d", "t"."e", "t"."f", "t"."g", 'foo' AS "remove_me" FROM "default"."T_EXCLUDE_TOP_LEVEL" AS "t" WHERE "t"."a") AS "subq";
---
--- -- EXCLUDE top-level columns with JOIN
--- --#[exclude-42]
--- SELECT "t1"."b", "t1"."d", "t1"."f", "t2"."a", "t2"."c", "t2"."e", "t2"."g" FROM "default"."T_EXCLUDE_TOP_LEVEL" AS "t1" INNER JOIN "default"."T_EXCLUDE_TOP_LEVEL" AS "t2" ON true;
---
--- -- EXCLUDE top-level columns with JOIN and WHERE
--- --#[exclude-43]
--- SELECT "t1"."b", "t1"."d", "t1"."f", "t2"."a", "t2"."c", "t2"."e", "t2"."g" FROM "default"."T_EXCLUDE_TOP_LEVEL" AS "t1" INNER JOIN "default"."T_EXCLUDE_TOP_LEVEL" AS "t2" ON true WHERE "t1"."a" AND "t2"."a";
---
--- -- EXCLUDE top-level columns with JOIN and WHERE and specified SELECT element
--- --#[exclude-44]
--- SELECT "t1"."b", "t1"."d", "t1"."f" FROM "default"."T_EXCLUDE_TOP_LEVEL" AS "t1" INNER JOIN "default"."T_EXCLUDE_TOP_LEVEL" AS "t2" ON true WHERE "t1"."a" AND "t2"."a";
---
--- -- EXCLUDE top-level columns with JOIN and WHERE and specified SELECT elements
--- --#[exclude-45]
--- SELECT "t1"."b", "t1"."d", "t1"."f", "t2"."a" AS "special" FROM "default"."T_EXCLUDE_TOP_LEVEL" AS "t1" INNER JOIN "default"."T_EXCLUDE_TOP_LEVEL" AS "t2" ON true WHERE "t1"."a" AND "t2"."a";
---
--- -- EXCLUDE top-level columns with multiple JOINs
--- --#[exclude-46]
--- SELECT
---     "t1"."a",
---     "t2"."b",
---     "t3"."c",
---     "t4"."d",
---     "t5"."e",
---     "t6"."f",
---     "t7"."g"
--- FROM
---     "default"."T_EXCLUDE_TOP_LEVEL" AS "t1" INNER JOIN
---     "default"."T_EXCLUDE_TOP_LEVEL" AS "t2" ON "t2"."a" LEFT OUTER JOIN
---     "default"."T_EXCLUDE_TOP_LEVEL" AS "t3" ON "t3"."a" INNER JOIN
---     "default"."T_EXCLUDE_TOP_LEVEL" AS "t4" ON "t4"."a" RIGHT OUTER JOIN
---     "default"."T_EXCLUDE_TOP_LEVEL" AS "t5" ON "t5"."a" FULL OUTER JOIN
```

Comment:
self-review: this prior test was wrong. the correct output should not have included the `OUTER`